examples/pytorch/Logging/pytorch log model.ipynb

```diff
@@ -0,0 +1,416 @@
+    "\n",
+    "In this guide we will:\n",
+    "- define a simple pytorch model\n",
+    "- set a model signature for our logged model to define inputs and outputs to the mlflow model\n",
+    "- create an mlflow experiment, where we will log runs and their corresponding models\n",
+    "- load the model back from storage, to use in other notebooks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Install dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "885e8968-cc18-48db-af92-1b034c6ea3af",
+     "showTitle": true,
+     "title": "Install dependencies"
+    },
+    "jupyter": {
+     "outputs_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%pip install mlflow torch\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Imports + data preparation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "092ae65d-cebd-4778-8db5-400097966a3e",
+     "showTitle": true,
+     "title": "Imports + data preparation"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "import torch.nn as nn\n",
+    "import torch.optim as optim\n",
+    "from sklearn.datasets import load_iris\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.preprocessing import StandardScaler\n",
+    "from torch.utils.data import DataLoader, TensorDataset\n",
+    "import mlflow\n",
+    "import mlflow.pytorch\n",
+    "\n",
+    "# Load and preprocess the Iris dataset\n",
+    "iris = load_iris()\n",
+    "X = iris.data\n",
+    "y = iris.target\n",
+    "\n",
+    "# Standardize features\n",
+    "scaler = StandardScaler()\n",
+    "X_scaled = scaler.fit_transform(X)\n",
+    "\n",
+    "# Split the data into training and testing sets\n",
+    "X_train, X_test, y_train, y_test = train_test_split(X_scaled, y, test_size=0.2, random_state=42)\n",
+    "\n",
+    "# Convert arrays to PyTorch tensors\n",
+    "X_train_tensor = torch.tensor(X_train, dtype=torch.float32)\n",
+    "y_train_tensor = torch.tensor(y_train, dtype=torch.long)\n",
+    "X_test_tensor = torch.tensor(X_test, dtype=torch.float32)\n",
+    "y_test_tensor = torch.tensor(y_test, dtype=torch.long)\n",
+    "\n",
+    "# Create datasets and dataloaders\n",
+    "train_dataset = TensorDataset(X_train_tensor, y_train_tensor)\n",
+    "train_loader = DataLoader(dataset=train_dataset, batch_size=16)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Define your pytorch model"
```

Comment:
It's a bit strange to jump from L1 title to L4 title directly, there are two options:
- use normal text.
- Use L2 title + some explanation on what your model is.